docker/kubernetes-agent-tentacle/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# The default image is bullseye-slim (Debian 11)
+ARG RuntimeDepsTag="6.0-bullseye-slim"
```

Comment:
This needs to be defined at the very top as a 'global' ARG because this Dockerfile uses a multi-stage build. Each `FROM` statement declares a new scope which lasts until either the next FROM or the end of the file.